### PR TITLE
MODDICORE-45: DI process finishes with ERROR status

### PR DESCRIPTION
**Pull request by VRohach**

### CODE REVIEW

File: ramls/instance.json [hunk 1]
```diff
@@ -310,7 +310,7 @@
         "properties": {
           "instanceNoteTypeId": {
             "description": "ID of the type of note",
-            "$ref": "raml-storage/schemas/common/uuid.json"
+            "$ref": "uuid.json"
```
Comment (KaterynaSenchenko) [inline ramls/instance.json:313]: are you sure this is the right path?

File: ramls/instance.json [hunk 2]
```diff
@@ -369,140 +369,140 @@
...
-        },
-        "additionalProperties": false
-      }
+        }
+      },
+      "additionalProperties": false
     }
   },
-  "additionalProperties": true,
+  "additionalProperties": false,
```
Comment (KaterynaSenchenko) [inline ramls/instance.json:505]: I have a feeling that we set it to true intentionally, better leave it this way

File: src/main/java/org/folio/processing/events/EventManager.java [hunk 3]
```diff
@@ -53,10 +59,12 @@ private EventManager() {
               if (publishThrowable == null) {
                 future.complete(eventPayload);
               } else {
+                LOGGER.error("Can`t publish event");
```
Comment (KaterynaSenchenko) [inline src/main/java/org/folio/processing/events/EventManager.java:62]:
```suggestion
                LOGGER.error("Can`t publish event", publishThrowable);
```

File: src/main/java/org/folio/processing/events/EventManager.java [hunk 4]
```diff
@@ -23,11 +24,16 @@
...
 
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
 /**
  * The central class to use for handlers registration and event handling.
  */
 public final class EventManager {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```
Comment (KaterynaSenchenko) [inline src/main/java/org/folio/processing/events/EventManager.java:35]:
```suggestion
  private static final Logger LOGGER = LoggerFactory.getLogger(EventManager.class);
```

Comment (VRohach) [reply src/main/java/org/folio/processing/events/EventManager.java]: Done

File: src/main/java/org/folio/processing/events/EventManager.java (hunk 3 above)
Comment (VRohach) [reply src/main/java/org/folio/processing/events/EventManager.java]: Done

**Pull request merged**